### ISSUE TITLE
Disable SITE_URL popup when admin logs in 

### ISSUE THREAD

**Issue by lunitic** [RocketChat/Rocket.Chat#2912]
Your Rocket.Chat version: 0.26

I understand the purpose of the SITE_URL popup but it can make some serious issue if the person who gets it doesn't know what it means. The human behavior is to click yes which can make the server unreachable.

My workaround is that I have made a 2nd admin role with the same permission as admin. But I think a "lock" site url would be much better.

**Comment by k0nsl** [comment]
Yes, agreed. This can be rather confusing for some administrators and one has to complete a "debriefing" on that issue for new administrators, or else we'll find ourselves in trouble.

I have my instance on a normal domain and also on Tor (.onion) so when somebody accesses the instance via the .onion URL they'll get that popup you mention..
